src/coverlet.core/Helpers/FileSystem.cs

```diff
@@ -3,11 +3,46 @@
 
 namespace Coverlet.Core.Helpers
 {
-    internal class FileSystem : IFileSystem
+    public class FileSystem : IFileSystem
```

Comment:
keep internal as interface

Reply:
I need to expose it to console and msbuild.task.

Reply:
ditto ok for now